lib/backburner/utils.ts

```diff
@@ -3,7 +3,7 @@ const NUMBER = /\d+/;
 export const now = Date.now;
 
 export function each<T>(collection: T[], callback: (v: T) => void) {
-  for (let i = 0; i < collection.length; i++) {
+  for (let i = 0, l = collection.length; i < l; i++) {
     callback(collection[i]);
```

Comment:
I'm not sure this optimization is required. Modern JITs are smart enough to figure this one out if it is possible, and if it isn't it will end up introducing the check (out of bounds) behind the scenes anyways